db/migrate/20180307093314_create_organizations.rb

```diff
@@ -0,0 +1,19 @@
+class CreateOrganizations < ActiveRecord::Migration[5.0]
+  def change
+    create_table :organizations do |t|
+      t.string :name, null: false
+
+      t.timestamps
+    end
+    add_column :teams, :organization_id, :integer, foreign_key: true, null: true
+    add_index :organizations, :name, unique: true
+
+    default_organization = Organization.find_or_create_by(name: 'Default organization')
```

Comment:
Ik zou zeggen Organization.create!
Volgens mij is het onmogelijk dat er al een organization bestaat omdat je letterlijk hierboven de organization table aanmaakt. En met een uitroepteken geeft het een error en worden je migraties teruggedraaid als er iets fout gaat.